app/views/spotlight/about_pages/_empty.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Building this about page</h2>
```

Comment:
@ggeisler should this page have any content (heading or other) for users that cannot edit the page?